astra/src/main/java/com/slack/astra/logstore/search/SearchResultUtils.java

```diff
@@ -683,6 +685,25 @@ public static SearchQuery fromSearchRequest(AstraSearch.SearchRequest searchRequ
       }
     }
 
+    AggregatorFactories.Builder aggregatorFactoriesBuilder = null;
+    if (!searchRequest.getAggregationJson().isEmpty()) {
+      SearchModule searchModule = new SearchModule(Settings.EMPTY, List.of());
+      try {
+        NamedXContentRegistry namedXContentRegistry =
```

Comment:
Can the search module / namedXContentRegistry/jsonXContentParser all be initialized once as well with the ObjectMapper?